jumpscale/sals/marketplace/deployer.py

```diff
@@ -216,7 +216,7 @@ def ask_multi_pool_distribution(
         messages = {f"Pool: {p} CU: {pools[p][0]} SU: {pools[p][1]}": p for p in pools}
         while True:
             pool_choices = bot.multi_list_choice(
-                f"Please seclect the pools you wish to distribute you {workload_name} on",
+                f"Please select the pools you wish to distribute your{workload_name} on",
```

Comment:
there will be no space between your and {workload_name}

Reply:
I updated. thank you